models/five_fold_validation.py

```diff
@@ -0,0 +1,41 @@
+from logistic_regression import LogisticRegression
+import pandas as pd
+import numpy as np
+
+
+def validate_logistic_regression_for_wine_quality():
+    num_of_folds = 5
+    learning_rate= 0.000001
+    max_iterations = 100
+    df = pd.read_csv("../data/winequality/winequality-red.csv", sep=";")
+    df['classified']=[1 if x>=6 else 0 for x in df["quality"] ]
+    fold_size = int(round(df.shape[0]/num_of_folds))
+    for i in range(5):
```

Comment:
update the range to fold_size

Reply:
actually i need to change it to num_of_folds

Reply:
yea num of folds